angular_components/1-base/pubspec.yaml

```diff
@@ -19,3 +19,7 @@ transformers:
     resolved_identifiers:
       Random: 'dart:math'
 - dart_to_js_script_rewriter
+
```

Comment:
I thought that our goal was to do this in steps, the first step being to _allow_ the use of dartdevc in `debug` mode (only). Doing so does not invalidate instructions describing the use of Dartium.